GameServerApp/Program.cs

```diff
@@ -17,13 +17,17 @@ public class ArgsOptions
         [Option("config", DefaultValue = "Settings/GameInfo.json")]
         public string ConfigPath { get; set; }
 
+        [Option("configJSON", DefaultValue = "")]
+        public string ConfigJSON { get; set; }
```

Comment:
I'd vote for calling this `ConfigJson` as well. Naming conventions regarding acronyms are still unaddressed in our guidelines, but to avoid conflicting rules with constants and to keep the names more consistent, we probably want to stick to pascal case and camel case, even with acronyms.